audb/core/load.py

```diff
@@ -1009,9 +1009,9 @@ def load_media(
         ...     format='flac',
         ...     verbose=False,
         ... )
-        >>> cache_root = audb.default_cache_root()
-        >>> [p[len(cache_root):] for p in paths]
-        ['/emodb/1.1.1/40bb2241/wav/03a01Fa.flac']
+        >>> path = os.path.normpath(paths[0])
```

Comment:
I guess the better option would be to use `os.path.normpath()` already inside `audb.load_media()` and at all places where we return an absolute path, so that it will not read `D:\a\b\c/d.wav` under Windows.

The same is true for the entries in the dataframes when we add `db.root`.

But it's also fine with me to handle this in another pull request as here the main idea was to add a test for Windows.

Reply:
I have now updated this and use `os.path.normpath` to return the correct full path.